packages/terra-table/src/SelectableTableRows.jsx

```diff
@@ -80,7 +163,6 @@ class SelectableTableRows extends React.Component {
   newPropsForRow(row, index, onClick, onKeyDown) {
     const isSelected = this.props.selectedIndexes.indexOf(index) >= 0;
     const newProps = { };
-
     // Set the isSelected attribute to false for all the rows except the rows whose index is set to state selectedIndex.
     if (isSelected !== row.props.isSelected) {
```

Comment:
This might be not related to code changes of this PR but I have doubt with this condition 
``` (isSelected !== row.props.isSelected) ```  
with this condition newProps.isSelected will be set to true only when the either of props in above condition is true but this will not set newProps.isSelected to true when both `isSelected` and `row.props.isSelected` are true.  is this is intended to be like this, Because i have seen this scenario in multi-selectable rows table where both `isSelected` and `row.props.isSelected` are true and newProps.isSelected is set to false.

This scenario happens when we have pre-selected rows in multi selectable rows table.  where both `isSelected` and `row.props.isSelected` will become true.  Since selectedIndexes will have index of pre-selected row and props of row too will have true, so in this scenario we might miss-out to update newProps.isSelected to true. 

Changing the above condition to  ``` if(isSelected || row.props.isSelected) ``` will update the newProps.isSelected 
to true on both cases like when either of them are true and when both of them are true.

Reply:
I'll try again if I get the chance, but I don't think row.props.isSelected would work. If you initialize a table row such as <Table.row isSelected />, that if condition would always be true, even if the row is no longer selected. We will need to solely rely on the selectedIndexes prop passed into the table, which from implementing the details I noticed refers to the previous state of the table.